cybersyn/scripts/central-planning.lua

```diff
@@ -9,6 +9,36 @@ local band = bit32.band
 local table_remove = table.remove
 local random = math.random
 
+local HASH_STRING = "CS_QUALITY_SEP"
```

Comment:
Isn't a simple pipe `|` safe? item names and quality cannot contain that, I think.

Reply:
Adressed in c12fddc977a8ba6ac3fc102888be4c8c0013c9f2

That's even better than a magic string. Wasn't originally sure about the restrictions on item names.